src/main/java/calculator/StringCalculator.java

```diff
@@ -0,0 +1,63 @@
+package calculator;
+
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class StringCalculator {
+
+	public static final String NEGATIVEDELIMITER = "-";
+	public static final String DELIMITERS = ",|:";
+	public static final String CUSTOMDELIMITER = "//(.)\n(.*)";
+
+	public static int cal(String input) {
+		if (isNullOrBlank(input)) {
+			return 0;
+		}
+
+		if (isContainNegative(input)) {
+			throw new RuntimeException();
+		}
+
+//		if (iscontainDelimiters(input)) {
+//			return sumInput(removeDelimiter(input, DELIMITERS));
+//		}
+//
+//		Matcher m = Pattern.compile(CUSTOMDELIMITER).matcher(input);
+//		if (m.find()) {
+//			String customDelimiter = m.group(1);
+//			return sumInput(removeDelimiter(m.group(2), customDelimiter));
+//		}
+
+		return sumInput(removeDelimiter(input));
+	}
+
+	private static String[] removeDelimiter(String input) {
+		Matcher m = Pattern.compile(CUSTOMDELIMITER).matcher(input);
+		if (m.find()) {
+			String customDelimiter = m.group(1);
+			String tokens[] = m.group(2).split(customDelimiter);
+			return tokens;
```

Comment:
```suggestion
			return m.group(2).split(customDelimiter);
```
변수에 담지말고 바로 리턴해도 됩니다 😄

Reply:
지역변수로 다시 쓰일 일이 없어서 바로 리턴하는게 좋은걸까요?

Reply:
[블로그](https://velog.io/@phjppo0918/%EB%B3%80%EC%88%98-%EC%9D%B8%EB%9D%BC%EC%9D%B8%ED%95%98%EA%B8%B0-Inline-Variable)에서도 잘 설명하고 있으니 한번 읽어보시면 도움될 것 같네요

intellij를 사용하고 있다면 inline variable 표시도 해줍니다.